.travis.yml

```diff
@@ -42,8 +42,8 @@ addons:
     sources:
       - ubuntu-toolchain-r-test
     packages:
-      - gcc-4.8
-      - g++-4.8
+      - gcc-6
+      - g++-6
```

Comment:
So it seems to build fine with gcc-5 in the test node, which is a more reasonable compiler for required dependency.